main.py

```diff
@@ -46,7 +46,8 @@
 class NoClientNetworkBlocksFound(Exception):
   def __init__(self, provider_name):
     Exception.__init__(self,
-                       'Could not find IP blocks associated with client provider {client_provider}.'.format(
+                       ('Could not find IP blocks associated with client ' +
+                       'provider {client_provider}.').format(
```

Comment:
The hanging indents for a lot of these changes aren't consistent with the style guide. See https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Indentation#Indentation